ajax/libs/material-components-web/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "material-components-web",
+  "version": "0.11.1",
+  "filename": "material-components-web.min.js",
+  "description": "Modular and customizable Material Design UI components for the web",
+  "homepage": "https://material.io/components/",
+  "keywords": [
+    "material components",
+    "material design"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/material-components/material-components-web.git"
+  },
+  "license": "Apache-2.0",
+  "author": "Travis Kaufman <@traviskaufman>",
```

Comment:
@traviskaufman OK, I see.
I really sorry for this. Would you add the "author" field in https://github.com/material-components/material-components-web/blob/master/package.json ?
We will follow this `material-components/material-components-web` repo of `package.json`'s contain to improve the `"author"` field.